controllers/topology/internal/mergepatch/mergepatch.go

```diff
@@ -42,6 +44,8 @@ type Helper struct {
 
 	// patch holds the merge patch in json format.
 	patch []byte
+
+	diff string
```

Comment:
Might be we want to exclude diff from this iteration given that the current output is hard to read and the diff compute logic needs some refinements

Reply:
Removed